include/library/spdm_crypt_lib.h

```diff
@@ -421,9 +447,17 @@ void libspdm_asym_free(uint32_t base_asym_algo, void *context);
 bool libspdm_asym_verify(
     spdm_version_number_t spdm_version, uint8_t op_code,
     uint32_t base_asym_algo, uint32_t base_hash_algo,
-    void *context, const uint8_t *message,
-    size_t message_size, const uint8_t *signature,
-    size_t sig_size);
+    void *context,
+    const uint8_t *message, size_t message_size,
+    const uint8_t *signature, size_t sig_size);
+
+bool libspdm_asym_verify_ex(
```

Comment:
Introducing "_ex" functions which have an additional "flags" parameter, which you can use to pass the BE Only\LE Only\Both.